source/common/network/listen_socket_impl.h

```diff
@@ -26,7 +26,12 @@ class SocketImpl : public virtual Socket {
       fd_ = -1;
     }
   }
-  void setOptions(const OptionsSharedPtr& options) override { options_ = options; }
+  void setOption(const OptionSharedPtr& option) override {
```

Comment:
looks like what it does is `addOption`, the interface and comment should be fixed.

Reply:
I'll change this.